### PR TITLE
[zero] add L2 gradient clipping for ZeRO 

### PR DESCRIPTION
**Pull request by 1SAA**

### CODE REVIEW

**Review by FrankLeeeee — APPROVED** [review]

File: tests/test_gemini/update/test_grad_clip.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        label = label.cuda()
+
+        zero_optim.zero_grad()
+        torch_optim.zero_grad()
+
+        torch_loss = run_fwd_bwd(torch_model, data, label, criterion, torch_optim)
+        loss = run_fwd_bwd(model, data, label, criterion, zero_optim)
+        assert_close(torch_loss, loss)
+
+        import apex.amp as apex_amp
```
Comment (feifeibear) [inline tests/test_gemini/update/test_grad_clip.py:94]: 我们没有强制需要装apex吧？
